### PR TITLE
Transaction detail #369

### LINKED ISSUE #369: Update transaction page design
Update transaction page design to reflect the new direction settled on in #231.

### PR DESCRIPTION
**Pull request by katibest**
Resolves #369 

## Changelog

* Updated transaction detail over page to new design 
* Updated logs tab with new design for individual log events
  

screen shot 2018-07-24 at 11 06 12 am
screen shot 2018-07-24 at 11 06 25 am

Multiple Logs:
screen shot 2018-07-24 at 2 15 29 pm

### CODE REVIEW

File: apps/explorer_web/test/explorer_web/features/viewing_transactions_test.exs [hunk 1]
```diff
@@ -135,19 +135,6 @@ defmodule ExplorerWeb.ViewingTransactionsTest do
...
-      transaction =
-        :transaction
-        |> insert(to_address: nil)
-        |> with_block()
-
-      internal_transaction = insert(:internal_transaction_create, transaction: transaction, index: 0)
-
-      session
-      |> TransactionPage.visit_page(transaction.hash)
-      |> assert_has(TransactionPage.contract_creation_address_hash(internal_transaction))
```
Comment (feliperenan) [inline apps/explorer_web/test/explorer_web/features/viewing_transactions_test.exs]: Can we remove `TransactionPage.contract_creation_address_hash/1`?, if I'm not wrong, here is the only place that was using it.

File: apps/explorer_web/lib/explorer_web/templates/transaction/overview.html.eex [hunk 2]
```diff
@@ -1,203 +1,111 @@
...
+          <span class="text-muted">
+            <%= gettext "Limit" %>
+            <%= format_gas_limit(@transaction.gas) %>
+          </span>
         </div>
       </div>
     </div>
   </div>
-</section>
+  </section>
```
Comment (feliperenan) [inline apps/explorer_web/lib/explorer_web/templates/transaction/overview.html.eex]: Wrong indentation here 🙈

File: apps/explorer_web/lib/explorer_web/templates/transaction_internal_transaction/index.html.eex [hunk 3]
```diff
@@ -1,83 +1,72 @@
...
+                    to: transaction_log_path(@conn, :index, @conn.assigns.locale, @transaction),
+                    "data-test": "transaction_logs_link"
+                  ) %>
+            </li>
+          </ul>
+
+          <!-- MOBILE DROPDOWN NAV -->
+          <ul class="nav nav-tabs card-header-tabs d-md-none">
+            <li class="nav-item dropdown flex-fill text-center">
+              <a class="nav-link active dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Internal Transactions</a>
```
Comment (feliperenan) [inline apps/explorer_web/lib/explorer_web/templates/transaction_internal_transaction/index.html.eex]: `gettext("Internal Transactions")` ?

File: apps/explorer_web/lib/explorer_web/templates/transaction/overview.html.eex (hunk 2 above)
Comment (katibest) [reply apps/explorer_web/lib/explorer_web/templates/transaction/overview.html.eex]: My bad!! Updated!

File: apps/explorer_web/lib/explorer_web/templates/transaction_internal_transaction/index.html.eex (hunk 3 above)
Comment (katibest) [reply apps/explorer_web/lib/explorer_web/templates/transaction_internal_transaction/index.html.eex]: Updated!

File: apps/explorer_web/test/explorer_web/features/viewing_transactions_test.exs (hunk 1 above)
Comment (katibest) [reply apps/explorer_web/test/explorer_web/features/viewing_transactions_test.exs]: You are correct Removed them. Good catch! 👍

**Comment by katibest** [comment]
@acravenho ok! So for this card we moved the value so it would not wrap. Its a bigger problem thats going to take a little more time to solve and we don't want to stop this code any longer from going through.  We also need to refactor the transaction tiles with tokens so I think we should make a new card for those two things.

**Comment by acravenho** [comment]
@katibest I'm a little disappointed that this can't be resolved now. I'm going to open another issue now regarding these decimals so we can get this merged.

**Comment by katibest** [comment]
@acravenho  We're disappointed too. Its definitely a hard problem to solve. Truncating the number but not "POA" is not as easy as it sounds, we may need some dev help. Since it's across the site and not just on transaction details I think making a new card makes more sense.

**Comment by acravenho** [comment]
@katibest No problem, we'll get it figured out. I created #471 for this issue.

**Pull request merged**